helm-charts/yunikorn/templates/configmap.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: yunikorn-configs
+  labels:
```

Comment:
Hi @yangwwei  
Under label, cud we add below items. So we will have same nomenclature across all yamls.
`    chart: {{ include "yunikorn.chart" . }}
    release: {{ .Release.Name }}
    heritage: {{ .Release.Service }}`

Reply:
Hi @sunilgovind , it makes sense, I just added that in the second commit, please take a look.